### PR TITLE
add toggle for tx type and integrate basefee metric

### PR DESCRIPTION
**Pull request by bford21**
1) Adds a toggle for pre/post EIP-1559 transaction types and only shows relevant data for that type
2) linting/formatting changes

Would be helpful to pull down and test locally. Changes are still "hidden" behind the `eip1559` query param

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -102,102 +102,121 @@ function gtag(){dataLayer.push(arguments);}
...
                       <div class="count fast">
-                        <?php echo ($gpTip['instant']) ?>
+                        <?php echo ($gpRecs2['fastest']/10) ?>
                       </div>
                       <div class="divider"></div>
                       <div class="text-container">
                         <div class="count_top"><span>trader</span><span>< ASAP</span></div>
+                        <!-- <div class="count_top usd">
+                          <?php // $fee = round($gpRecs2['fastest']*21000/1e9*$exchangeRate/10, 3); echo($currString . $fee . ' / Transfer'); ?>
+                        </div> -->
```
Comment (AndreMiras) [inline index.php]: Do we want to keep these commented out code or should we simply drop it?

**Review by AndreMiras — APPROVED** [review]
LGTM, tested in local successfully 
image

File: index.php (hunk 1 above)
Comment (bford21) [reply index.php]: Cleanup up these comments and also changed things to display the base fee in gwei. Rebased as well for cleaner commits

**Review by AndreMiras — APPROVED** [review]
LGTM

**Pull request merged**